docs/source/pages/installation.rst

```diff
@@ -36,12 +38,14 @@ To verify your setup, go to the directory containing the downloaded ``run_macaro
 
 .. code-block:: shell
 
-  ./run_macaron.sh --help
+  ./run_macaron.sh macaron --help
```

Comment:
No need to add `macaron` (similar in other places):
```suggestion
  ./run_macaron.sh --help
```

Reply:
Addressed in https://github.com/oracle-samples/macaron/pull/269/commits/fc12961486572c6e91d62fa014813a49bf96df07